cloudferrylib/os/image/glance_image.py

```diff
@@ -154,14 +154,13 @@ def get_image_status(self, image_id):
     def get_ref_image(self, image_id):
         try:
             return self.glance_client.images.data(image_id)._resp
-        except exc.HTTPInternalServerError:
+        except glance_exceptions.HTTPInternalServerError:
             raise exception.ImageDownloadError
 
     def get_image_checksum(self, image_id):
         return self.get_image_by_id(image_id).checksum
 
-    @staticmethod
```

Comment:
In this case we can use `resource` variable instead of providing `self` and getting rid of static method...